packages/autoprofile/precompile/docker-opts.js

```diff
@@ -16,10 +16,7 @@ module.exports = exports = function getDockerOpts(family, version) {
   const targetNodeVersion = version || nodeJsVersion;
 
   if (family === GLIBC) {
-    // NOTE: debian stretch was no longer working good with some Node versions
-    // https://stackoverflow.com/questions/76094428/debian-stretch-repositories-404-not-found
-    // https://stackoverflow.com/questions/52083380/in-docker-image-names-what-is-the-difference-between-alpine-jessie-stretch-an
-    baseImage = `node:${targetNodeVersion}-buster`;
+    baseImage = `node:${targetNodeVersion}`;
```

Comment:
Question: Why did you remove buster postfix?

Reply:
See commit comment:

> Also, use the Docker image node:${targetNodeVersion}
instead of node:${targetNodeVersion}-buster for building the autoprofile
binaries. Specifying a specific Debian distribution has a tiny
maintenance disadvantage, as node images are not always provided for all
Debian distributions. There is no node:21.2.0-buster, for example, as
Debian Buster is simply too old by now.

But if you want to bring them back, that's fine as well.

Reply:
My thinking was: The next time you folks update for, let's say Node.js 26, and the Node.js project has stopped providing a node.js-bullseye image, you need to update this again. I thought we might not need to be that specific and just using nodes image without postfix is fine as well.

Reply:
I have added a TODO to #884.